features-json/colr-v1.json

```diff
@@ -62,7 +62,8 @@
       "94":"n",
       "95":"n",
       "96":"n",
-      "97":"n"
+      "97":"n",
+      "98":"n"
```

Comment:
Needs to be removed in order for the checks to be passed. New browser versions can't be added via PRs